screens/Request/ReceiveVcScreen.tsx

```diff
@@ -83,11 +85,15 @@ export const ReceiveVcScreen: React.FC = () => {
         message={t('saving')}
         progress={true}
       />
-      <ErrorMessageOverlay
-        isVisible={controller.IsSavingFailedInIdle}
-        error={storeErrorTranslationPath}
-        translationPath={'ReceiveVcScreen'}
-        onDismiss={controller.DISMISS}
+
```

Comment:
@PuBHARGAVI please make changes to show Received Vc only after it is saved successfully. While saving the progess bar can be shown.